### PR TITLE
Wrong selector starts an infinite loop

### PR DESCRIPTION
**Pull request by kl3sk**
First of all, thanks for your work, it save my time :D

Because I custom my form or sometime the structure may change, the default selector 

eg```> div``` may fail.

So i add a security to prevent an infinite loop. The user will be prompted in the console, but this is just a proposal.

### CODE REVIEW

File: jquery.collection.js [hunk 1]
```diff
@@ -357,10 +357,16 @@
...
                 var container = $(settings.container);
                 var button = collection.find('.' + settings.prefix + '-add, .' + settings.prefix + '-rescue-add, .' + settings.prefix + '-duplicate').first();
+                var secure = 0;
                 while (elements.length < settings.init_with_n_elements) {
+                    secure++;
                     var element = elements.length > 0 ? elements.last() : undefined;
                     var index = elements.length - 1;
                     elements = doAdd(container, button, collection, settings, elements, element, index, false);
+                    if(secure > settings.init_with_n_elements) {
+                        console.error('Infinite loop, element selector (' + settings.elements_selector + ') not found !!')
```
Comment (ninsuo) [inline jquery.collection.js]: I guess you need to add a ";" at the end of this line.
And can you remove one ! as 2 looks a bit aggressive 😄

Comment (kl3sk) [reply jquery.collection.js]: Yes one "!" is enough ^^

For the ";", i'll use to not use it >< 
so it's a mistake (it is not mandatory in javascript)

Comment (ninsuo) [reply jquery.collection.js]: oh ok, didn't know about the optional `;` 👍 thanks for the info.

**Comment by ninsuo** [comment]
Thank you @kl3sk! 👍

**Comment by kl3sk** [comment]
@ninsuo Welcome

**Comment by ninsuo** [comment]
Yes it should be. 
I’m on a train right now so cant check but normally i pushed it

**Comment by kl3sk** [comment]
Ok still 2.1.25 on npmjs as now.

let's wait for it ;)

**Comment by ninsuo** [comment]
Image

Should be good now 🤓
Have a great weekend!

**Pull request merged**